### PR TITLE
Fix Target YuPiF4

### PR DESCRIPTION
**Pull request by ted-rcnet**
Hi,
Bug fix config gyro  Target

### CODE REVIEW

**Comment by ted-rcnet** [comment]
Hi,
Bug fix gyro config Target

File: src/main/target/YUPIF4/target.h [hunk 1]
```diff
@@ -22,76 +22,79 @@
...
+#define ICM20689_EXTI_PIN       PC4
+
+#define ACC
+#define USE_ACC_SPI_MPU6500
 #define ACC_MPU6500_ALIGN       CW90_DEG
 
-#define USE_GYRO
-#define USE_GYRO_MPU6500
+#define GYRO
+#define USE_GYRO_SPI_MPU6500
```
Comment (digitalentity) [inline src/main/target/YUPIF4/target.h]: This define is not necessary

File: src/main/target/YUPIF4/target.h [hunk 2]
```diff
@@ -22,76 +22,79 @@
...
 #define MPU6500_CS_PIN          SPI1_NSS_PIN
 #define MPU6500_SPI_BUS         BUS_SPI1
 
-#define USE_ACC
+#define USE_GYRO_MPU6500
 #define USE_ACC_MPU6500
+#define ICM20689_CS_PIN         SPI1_NSS_PIN
+#define ICM20689_EXTI_PIN       PC4
+
+#define ACC
```
Comment (digitalentity) [inline src/main/target/YUPIF4/target.h]: INAV uses concept of `USE_*` macros. This definition is now meaningless

File: src/main/target/YUPIF4/target.h [hunk 3]
```diff
@@ -22,76 +22,79 @@
...
 
 #define USE_I2C
 #define USE_I2C_DEVICE_2
-#define I2C2_SCL                PB10
-#define I2C2_SDA                PB11
+#define I2C1_SCL                PB10
+#define I2C1_SDA                PB11
 
-#define USE_MAG
+#define MAG
```
Comment (digitalentity) [inline src/main/target/YUPIF4/target.h]: Change to `USE_*`

**Review by digitalentity — CHANGES_REQUESTED** [review]
Please test on real hardware.

**Comment by ted-rcnet** [comment]
hi,
I've been testing on real hardware, I am the designer of the board.

**Comment by digitalentity** [comment]
@ted-rcnet ok, yet I think this PR was not fully tested on real hardware. With `target.h` file from the PR you should end up without compass at best.

**Comment by digitalentity** [comment]
See my comments to this PR

**Review by digitalentity — DISMISSED** [review]

**Comment by ted-rcnet** [comment]
Oops Sorry I didn't push the right target :/

**Comment by ted-rcnet** [comment]
Thanks Konstantin.

**Pull request merged**